app/src/main/java/org/fossasia/phimpme/accounts/AccountActivity.java

```diff
@@ -102,7 +104,8 @@
     private static final int OWNCLOUD_REQUEST_CODE = 9;
     private static final int RESULT_OK = 1;
     private static final int RC_SIGN_IN = 9001;
-    public final static String CLOUDRAIL_APP_KEY = "/*App Key */";
+    public static final String BROWSABLE = "android.intent.category.BROWSABLE";
+    public final static String CLOUDRAIL_APP_KEY = "59c956f8c8b6755b444e4595";//CloudRail_App-Key
```

Comment:
Please put this in constants.java file and remove your personal app key.

Reply:
@anantprsd5 Updated !